package-lock.json

```diff
@@ -0,0 +1,13089 @@
+{
```

Comment:
Can you delete this package-lock.json? We use yarn for our package installs. Having both a yarn.lock and a package-lock.json will break the app when deployed to Passsenger.

Reply:
Sure, sorry it was created while working with seeding the PostgreSQL by switching between PowerShell and WSL.